platform/kube/inject/inject.go

```diff
@@ -190,6 +193,20 @@ func GetInitializerConfig(kube kubernetes.Interface, namespace, injectConfigName
 		return nil, err
 	}
 
+	if c.IncludeNamespaces != nil && c.ExcludeNamespaces != nil {
```

Comment:
Please add additional test cases to TestGetInitializerConfig() for these new error cases

Reply:
Done